crates/ruff/tests/snapshots/integration_test__rule_f401.snap

```diff
@@ -34,16 +34,32 @@ marking it as unused, as in:
 from module import member as member
 ```
 
+Within `__init__.py` re-export a symbol with an `__all__` declaration containing a string
+matching the symbol, as in:
```

Comment:
I don't think you need to specify the `__init__.py` part, you can follow the above style.

> Alternatively, you can use `__all__` to declare a symbol as part of the module's public interface,
> as in: